### DISCUSSION TITLE
Aimbot

### DISCUSSION THREAD

**Discussion by jolbus52** [YimMenu/YimMenu#2737] [General]
Might be a bit obnoxious but its been a while since the aimbots been completely broken and would like some kind of update on the situation

**Comment by Yimura** [comment]
Have you checked any open issues? I'm sure you were able to find the solution to your question there (my standpoint on it) and other people their opinions on the matter as well as any possible solutions.

**Comment by gfdgfdfgd** [comment]
is there a new update or something? because when i inject it into gta it closes my gta

**Reply by Deadlineem** [reply]
Download the latest DLL from the main page.

**Reply by emree2910** [reply]
does not help
